website/docs/glue-catalog.md

```diff
@@ -149,6 +150,47 @@ From your terminal, run the glue crawler.
 Once the crawler succeeds, you’ll be able to query this Iceberg table from Athena,
 EMR and/or Redshift query engines.
 
+
+#### Method 2: Using XTable APIs to sync with AWS Glue Data Catalog directly
+This applies for iceberg target format only.
+
+**Pre-requisites:**
+* Download iceberg-aws-X.X.X.jar from the [Maven repository](https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-aws)
+* Download bundle-X.X.X.jar from the [Maven repository](https://mvnrepository.com/artifact/software.amazon.awssdk/bundle)
+
+Create a `glue-sync-config.yaml` file:
+
+```yaml md title="yaml"
+sourceFormat: HUDI
+targetFormats:
+   - ICEBERG
+datasets:
+   -
+      tableBasePath: s3://path/to/source/data
+      tableName: table_name
+      partitionSpec: partitionpath:VALUE
+      namespace: xtable_synced_db
+```
+
+Create a `glue-sync-catalog.yaml` file:
+
+```yaml md title="yaml"
+catalogImpl: org.apache.iceberg.aws.glue.GlueCatalog
+catalogName: xtable
+catalogOptions:
+   io-impl: org.apache.iceberg.aws.s3.S3FileIO
+   warehouse: s3://path/to/source
+```
+
+Sample command to sync the table with Glue Data Catalog:
+
+```shell md title="shell"
+java -cp /path/to/xtable-utilities-0.2.0-SNAPSHOT-bundled.jar:/path/to/iceberg-aws-1.3.1.jar:/path/to/bundle-2.23.9.jar org.apache.xtable.utilities.RunSync  --datasetConfig glue-sync-config.yaml --icebergCatalogConfig glue-sync-catalog.yaml
+```
+### Validating the results
+After the crawler runs successfully, you can inspect the catalogued tables in Glue
```

Comment:
[Clarification] I am not familiar with the Glue stack, so I want to confirm if a crawler is indeed involved here.